cmd/root_test.go

```diff
@@ -0,0 +1,20 @@
+package cmd
+
+import (
+	"bytes"
+	"regexp"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_ExecuteVersionCommand(t *testing.T) {
```

Comment:
Do we need the underscore `_` here? In the other test file we do not use, hence I would keep the same nomenclature if possible.

Reply:
Agreed, sorry for adding bad styling code :bow: 

How about renaming https://github.com/gennaro-tedesco/gh-i/pull/25/commits/84f59b0f53b5a935f8120ed595ebd74f7bea0fd2? I can squash these commits if required.

Reply:
Yes, let's squash the commits if possible and then we can merge.

Reply:
I have squashed all 3 commits in https://github.com/gennaro-tedesco/gh-i/pull/25/commits/938181edf33f25cf712420d8edd7fbe24e4487bd